Directory.Build.targets

```diff
@@ -85,4 +85,28 @@
       </AssemblyAttribute>
     </ItemGroup>
   </Target>
+
+  <!-- RestrictedInternalsVisibleTo -->
+  <ItemDefinitionGroup>
+    <RestrictedInternalsVisibleTo>
+      <Visible>false</Visible>
```

Comment:
What's this for?

Reply:
It probably doesn't have much impact, but it matches the item definition for `InternalsVisibleTo` in arcade:

https://github.com/dotnet/arcade/blob/4a7d983f833d6b86365ea1b2b4d6ee72fbdbf944/src/Microsoft.DotNet.Arcade.Sdk/tools/GenerateInternalsVisibleTo.targets#L8-L12